inference_perf/config.py

```diff
@@ -44,7 +44,7 @@ class Distribution(BaseModel):
     max: int = 1024
     mean: float = 512
     std_dev: float = 200
-    total_count: int = 1000
+    total_count: Optional[int] = None
```

Comment:
Might make sense to update the example profiles and remove total_count from there so people don't copy and use them as is.